modules/ossm-control-plane-deploy.adoc

```diff
@@ -46,3 +46,31 @@ $ oc create -n istio-system -f istio-installation.yaml
 ----
 $ oc get pods -n istio-system -w
 ----
+
+= Deploying the control plane using OperatorHub
+
+.Prerequisites
+
+* An account with maistra-admin access (needs to be confirmed).
+
+* The {ProductName} Operator must be installed.
+
+.Procedure
+
+* Create a new project in the OpenShift Container Platform web console for this procedure. This example uses a project called istio-system.
```

Comment:
Is there any requirement that the namespace be "istio-system"?

Reply:
Nope.